modules/openapi-generator/src/main/resources/kotlin-client/data_class_opt_var.mustache

```diff
@@ -18,4 +15,4 @@
     {{#deprecated}}
     @Deprecated(message = "This property is deprecated.")
     {{/deprecated}}
-    {{#multiplatform}}@SerialName(value = "{{{vendorExtensions.x-base-name-literal}}}") {{/multiplatform}}{{#isInherited}}override {{/isInherited}}{{>modelMutable}} {{{name}}}: {{#isArray}}{{#isList}}kotlin.collections.List{{/isList}}{{^isList}}kotlin.Array{{/isList}}<{{^items.isEnum}}{{^items.isPrimitiveType}}{{^items.isModel}}{{#kotlinx_serialization}}@Contextual {{/kotlinx_serialization}}{{/items.isModel}}{{/items.isPrimitiveType}}{{{items.dataType}}}{{/items.isEnum}}{{#items.isEnum}}{{classname}}.{{{nameInCamelCase}}}{{/items.isEnum}}>{{/isArray}}{{^isEnum}}{{^isArray}}{{{dataType}}}{{/isArray}}{{/isEnum}}{{#isEnum}}{{^isArray}}{{classname}}.{{{nameInCamelCase}}}{{/isArray}}{{/isEnum}}? = {{#defaultvalue}}{{defaultvalue}}{{/defaultvalue}}{{^defaultvalue}}null{{/defaultvalue}}
\ No newline at end of file
+    {{#multiplatform}}@SerialName(value = "{{{vendorExtensions.x-base-name-literal}}}") {{/multiplatform}}{{#isInherited}}override {{/isInherited}}{{>modelMutable}} `{{name}}`: {{#isArray}}{{#isList}}kotlin.collections.List{{/isList}}{{^isList}}kotlin.Array{{/isList}}<{{^items.isEnum}}{{^items.isPrimitiveType}}{{^items.isModel}}{{#kotlinx_serialization}}@Contextual {{/kotlinx_serialization}}{{/items.isModel}}{{/items.isPrimitiveType}}{{{items.dataType}}}{{/items.isEnum}}{{#items.isEnum}}{{classname}}.{{{nameInCamelCase}}}{{/items.isEnum}}>{{/isArray}}{{^isEnum}}{{^isArray}}{{{dataType}}}{{/isArray}}{{/isEnum}}{{#isEnum}}{{^isArray}}{{classname}}.{{{nameInCamelCase}}}{{/isArray}}{{/isEnum}}? = {{#defaultvalue}}{{{defaultvalue}}}{{/defaultvalue}}{{^defaultvalue}}null{{/defaultvalue}}
```

Comment:
@noordawod I'm not sure about this change. I would like to hear someone else on this change.

Reply:
If you have a property which is a reserved word in Kotlin (`internal`, `public`, `when`, etc.) then it won't be escaped automatically by openapi generator. I don't know why, but it just doesn't. So the simplest solution is one of two: 

1) Add an underscore to the end of each property (I believe it was like so in an earlier incarnation).
2) Surround with a backtick.
3) Fix the generator to escape reserved words based on language.

Am I missing something?

Reply:
This is related to https://github.com/OpenAPITools/openapi-generator/pull/9842
Do you miss any more words?

Reply:
`public`, `private` were not escaped.

Reply:
Could you please add those words to the reserved words list, and revert the back-ticks changes?

Reply:
I can do that. So basically, to touch the Java code, that's what you mean? (`modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractKotlinCodegen.java`)

Reply:
Yes please 👍

Reply:
Done!